site/ba.njk

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+meta_title: Pokret za zaustavljanje COVID-19 pandemije
+meta_description: Pridonesi borbi i spasi živote ne radeći nista! Jednostavno ostani jebeno kod kuće!
```

Comment:
We dont say "Jednostavno ostani jebeno kod kuće!". it's super awkward. "Jednostavno ostani u jebenoj kući!" is more natural.